tsMuxerGUI/tsmuxerwindow.cpp

```diff
@@ -459,6 +457,8 @@ TsMuxerWindow::TsMuxerWindow()
           &TsMuxerWindow::onGeneralCheckboxClicked);
   connect(ui.checkBoxBlankPL, &QPushButton::clicked, this,
           &TsMuxerWindow::onSavedParamChanged);
+  connect(ui.checkBoxV3, &CheckBox::stateChanged, this,

```

Comment:
You need too change &CheckBox to &QCheckBox, otherwise it fails to compile with error:
```
../tsMuxerGUI/tsmuxerwindow.cpp: In constructor ‘TsMuxerWindow::TsMuxerWindow()’:
../tsMuxerGUI/tsmuxerwindow.cpp:460:27: error: ‘CheckBox’ has not been declared
   connect(ui.checkBoxV3, &CheckBox::stateChanged, this,
                           ^~~~~~~~
```

Reply:
Indeed.... Done.